spec/std/yaml/yaml_pull_parser_spec.cr

```diff
@@ -95,5 +106,11 @@ module YAML
         end
       end
     end
+
+    assert_raw %(hello)
+    assert_raw %("hello"), %(hello)
+    assert_raw %(["hello"])
+    assert_raw %(["hello","world"])
+    assert_raw %({"hello":"world"})
```

Comment:
Nope, that doesn't work. I'll fix it to throw an error for the alias, those are not supported in the pull parser.